packages/@azure/servicebus/data-plane/lib/session/messageSession.ts

```diff
@@ -518,8 +507,8 @@ export class MessageSession extends LinkEntity {
     }
     if (!options) options = {};
     this._isReceivingMessages = true;
-    this.maxConcurrentCallsPerSession = options.maxConcurrentCallsPerSession || 1;
-    this.maxMessageWaitTimeoutInSeconds = options.maxMessageWaitTimeoutInSeconds;
```

Comment:
Did we get rid of this line by mistake? Seems like we are still using the maxMessageWaitTimeoutInSeconds property.